tools/vf-component-library/src/site/_includes/layouts/components.njk

```diff
@@ -246,6 +224,47 @@ import { {{ component.baseHandle | camelize(true) }} } from "@visual-framework/{
 
       {% markdown %}
 
+      {% if component.notes %}
+      {% set readmeUsage = component.notes | string | stringBetween("## Usage", "\n## ") %}
+      {% set readmeInstallation = component.notes | string | stringBetween("## Install", "\n## ") %}
+
+
+      {% if readmeUsage %}
+        <h2 id="usage">Usage</h2>
+        {{ readmeUsage | markdown | safe }}
+      {% endif %}
+
+      {% if readmeInstallation %}
+      <details class="vf-details">
+        <summary class="vf-details--summary" id="installation">Installation info</summary>
+        {{ readmeInstallation | replace("## Install", "") | markdown | safe }}
+      </details>
+      {% endif %}
+
+      {% if 'changelog.md'|length %}
+      {% markdown %}
+      <div class="vf-stack vf-stack--custom" style="--vf-stack-margin--custom: .5rem;">
+      <h3>Changelog</h3>
+      {% set readmeChangelog = 'changelog.md' | string | stringBetween("###", "\n### ") %}
+      {# {{ readmeChangelog | markdown | safe }} #}
+      <details class="vf-details" id="changelog">
+        <summary class="vf-details--summary">Changelog</summary>
+
+        {% include 'changelog.md' %}
```

Comment:
hah, it was this easy!

does this not explode for nested components like vf-form--checkbox?

Reply:
I'm thinking that we should extrapolate the form components into their own little page each now.

Reply:
as `vf-forms` is it's own component - we get the changelog for that, so it's not exploding

Reply:
Are you happy with this, so I can resolve this?